prow/resultstore/writer/writer.go

```diff
@@ -85,6 +105,9 @@ func New(ctx context.Context, log *logrus.Entry, client ResultStoreBatchClient,
 		inv, err := w.client.CreateInvocation(ctx, w.createInvocationRequest(invID, inv))
 		if err != nil {
 			log.Errorf("resultstore.CreateInvocation: %v", err)
+			if persistentError(err) {
```

Comment:
Nit: add a comment that like "Stop retrying if the error is a persistent one."

Reply:
Done